bowling/src/fileReader.cpp

```diff
@@ -0,0 +1,66 @@
+#include <algorithm>
+#include <fstream>
+
+#include "fileReader.hpp"
+
+namespace fs = std::experimental::filesystem;
```

Comment:
It would be even better to move the namespace definition to a .hpp file and replace all occurrences of 'std::experimental::filesystem' with 'fs' there (as it is done here in this .cpp file).

Reply:
@mariuszlisowski @m-kop 
I'm not sure if defining namespace in .hpp files is a good practise? Wouldn't this namespace propagate to any other file where this .hpp is included? Do we want/need it?

Reply:
So instead of 'namespcae' let's use 'using' keyword. That's what I originally meant.

Reply:
It's not a namespace definition. "fs" is alias for the namespace. See https://en.cppreference.com/w/cpp/language/namespace